tests/components/emulated_roku/test_binding.py

```diff
@@ -14,14 +15,15 @@
     ROKU_COMMAND_LAUNCH,
     EmulatedRoku,
 )
-from homeassistant.core import HomeAssistant
+from homeassistant.core import Event, HomeAssistant
 
 
 async def test_events_fired_properly(hass: HomeAssistant) -> None:
     """Test that events are fired correctly."""
-    binding = EmulatedRoku(
-        hass, "Test Emulated Roku", "1.2.3.4", 8060, None, None, None
-    )
+    random_name = uuid4().hex
+    # Note that this test is accessing the internal EmulatedRoku class
```

Comment:
This test is still accessing internals, but it would significantly increase the scope and size of this PR so I didn't fix this case